front/src/applications/stdcm/components/StdcmForm/StdcmVias.tsx

```diff
@@ -31,8 +30,12 @@ const StdcmVias = ({ disabled = false }: StdcmConfigCardProps) => {
   const intermediatePoints = useMemo(() => pathSteps.slice(1, -1), [pathSteps]);
 
   const updateStopType = (newStopType: StdcmStopTypes, pathStep: StdcmPathStep) => {
-    const defaultStopTime =
-      newStopType === StdcmStopTypes.DRIVER_SWITCH ? formatDurationAsISO8601(3 * 60) : '';
+    let defaultStopTime: number | undefined;
+    if (newStopType === StdcmStopTypes.DRIVER_SWITCH) {
+      defaultStopTime = 3;
+    } else if (newStopType === StdcmStopTypes.SERVICE_STOP) {
+      defaultStopTime = 0;
+    }
```

Comment:
Looks like the store is updated but not the form in the UI

Reply:
Done